### PR TITLE
Designed the landing page

### PR DESCRIPTION
**Pull request by SoniGarima**
The PR is for issue #57 
I have added the following:
1. Logo
2. Slogan
3. Names of all the classifiers included in this project ( Can add more later)
4. [Video](https://drive.google.com/file/d/1EkHoW10z5WJbptyyZpnVMhd73R4HzLve/view?usp=sharing) illustration on how to use this web application

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -46,56 +45,71 @@ def main():
         
     if choice == "Home":
```
Comment (smaranjitghose) [inline app.py:46]: By default Home should be the first thing we visit when the app loads as well as in the Menu Bar on the left

File: app.py [hunk 2]
```diff
@@ -46,56 +45,71 @@ def main():
...
+        heart_img = Image.open('assets/Heart-graphic.jpg')
+        heart_img = heart_img.resize((100,100))
+        pneu_img = Image.open('assets/pneumonia.jpg')
+        pneu_img = pneu_img.resize((100,100))
+        skin_img = Image.open('assets/skin-cancer.jpg')
+        skin_img = skin_img.resize((100,100))
+        st.image(img)
+        st.image([covid_img,diabetic_img,heart_img,pneu_img,skin_img],caption=["COVID-19 Detection","Diabetic Retinopathy","Heart Disease Prediction","Pneumonia Detection","Skin Cancer Detection"])
+        st.write("Illustration of the Web Application:")
+        st.video('assets/demo.mp4')
```
Comment (smaranjitghose) [inline app.py:62]: Can you please put a small recording of the app with this change in the description of this Pull Request

File: app.py [hunk 3]
```diff
@@ -46,56 +45,71 @@ def main():
...
+        
+# below is the sample function you need to implement in separate python file
+
```
Comment (smaranjitghose) [inline app.py:65]: Simply remove the code. If required we can define it in another script and import it

**Review by smaranjitghose — APPROVED** [review]

File: app.py (hunk 1 above)
Comment (SoniGarima) [reply app.py:46]: Then what change should be done?

File: app.py (hunk 2 above)
Comment (SoniGarima) [reply app.py:62]: Yes, [Here](https://drive.google.com/file/d/1EkHoW10z5WJbptyyZpnVMhd73R4HzLve/view?usp=sharing) is the link.

File: app.py (hunk 3 above)
Comment (SoniGarima) [reply app.py:65]: Okay.

File: app.py (hunk 1 above)
Comment (smaranjitghose) [reply app.py:46]: The first if statement should be for Home Page

File: app.py (hunk 2 above)
Comment (smaranjitghose) [reply app.py:62]: Thanks

File: app.py (hunk 1 above)
Comment (SoniGarima) [reply app.py:46]: okay.

**Comment by SoniGarima** [comment]
I have done the required changes, could you please review?

**Review by smaranjitghose — APPROVED** [review]

**Comment by smaranjitghose** [comment]
give @SoniGarima 150 points

**Pull request merged**